### ISSUE TITLE
Phasespace plugin uses openPMD output

### ISSUE THREAD

**Issue by finnolec** [ComputationalRadiationPhysics/picongpu#3907]
Hello,

I'm currently trying to get PIConGPU running on `taurus` and I ran into some trouble because my `LaserWakefield` examples with the default `cfg` always kept crashing with the error message 
```
Unhandled exception of type 'St13runtime_error' with message 'openPMD-api built without HDF5 support', terminating
Unhandled exception of type 'St13runtime_error' with message 'openPMD-api built without HDF5 support', terminating
Unhandled exception of type 'St13runtime_error' with message 'openPMD-api built without HDF5 support', terminating
Unhandled exception of type 'St13runtime_error' with message 'openPMD-api built without HDF5 support', terminating
[taurusi2053:28657:0:28657] Caught signal 11 (Segmentation fault: Sent by the kernel at address (nil))
```
**I have already found the cause for this crash:** The phasespace plugin uses openPMD HDF5 output as default output although I set the regular openPMD backend to adios2 in the cfg with `--openPMD.ext bp`. I just assumed that I had a corrupted openPMD installation (one has to manually install it) instead of checking for the dependencies from every single plugin. So in case you also get this error message, you should look for openPMD dependencies within activated plugins and change the backend respectively.

So the 3 reasons for this issue are:
* Make the problem google-able in case someone else has a similar crash on another system which doesn't have a prebuilt openPMD-api module
* Is it possible to have a clearer error message which shows the responsible plugin?
* Would it be smart to change the backend for openPMD output from **all** plugins who use openPMD **globally** for the simulation so that one doesn't have to choose it manually in case no hdf5 support is installed? Maybe all openPMD dependent plugins could check for the `--openPMD.ext <backend>` parameter and use this as their default if no plugin specific backend is chosen.

**Comment by psychocoderHPC** [comment]
Offline discussed with @finnolec @steindev Plugins are independent of each other and it is hard to have an enforced global default. We could say that all plugins should use adios as the default openPMD backend if available. 
At the moment the radiation plugin is not able to use openPMD/adios because the tools are not ported to use openPMD-api.

**Comment by sbastrakov** [comment]
I guess we could make a new value for `.ext` option, so that it picks whatever (h5, bp) is available in the openPMD API version used? And then have it as default for all such plugins

**Comment by finnolec** [comment]
> I guess we could make a new value for `.ext` option, so that it picks whatever (h5, bp) is available in the openPMD API version used? And then have it as default for all such plugins

This sounds good

**Comment by sbastrakov** [comment]
Solved with #4155.
